edc-extensions/ssi/ssi-identity-core/src/main/java/org/eclipse/tractusx/edc/iam/ssi/identity/SsiIdentityServiceExtension.java

```diff
@@ -29,6 +31,9 @@ public class SsiIdentityServiceExtension implements ServiceExtension {
 
     public static final String EXTENSION_NAME = "SSI Identity Service";
 
+    @Setting(value = "SSI Endpoint audience")
```

Comment:
Will not work with `participant-id` since for know it's not passed in the `TokenParamenter` to the identity service. We can only use for now the connector address